.php-cs-fixer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = (new PhpCsFixer\Finder())
+    ->files()
+    ->in([__DIR__.'/app', __DIR__.'/tests', __DIR__.'/config', __DIR__.'/database', __DIR__.'/vendor/biigle'])
+    ->name('*.php')
+    ->ignoreDotFiles(true)
+    ->ignoreVCS(true);
+
+return (new PhpCsFixer\Config())
+    ->setRules([
+        '@PSR2' => true,
+        'ordered_imports' => true,
+        'single_space_after_construct' => true,
+        'single_quote' => true,
+        'array_indentation' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'combine_consecutive_unsets' => true,
+        'class_attributes_separation' => ['elements' => ['method' => 'one', ]],
+        'no_whitespace_before_comma_in_array' => true,
+        'unary_operator_spaces' => true,
+        'whitespace_after_comma_in_array' => true,
+        'method_chaining_indentation' => true,
```

Comment:
This one always bugged me. On the one hand, method chains should be properly indented. On the other hand I have several chains that I would like to indent _differently_ for better readability. I always had to revert these in the past when I used the CS fixer. Maybe we can find a proper ruleset for this. See also my review comment above.